hack/deploy-karmada.sh

```diff
@@ -21,9 +21,15 @@ CLUSTER_IP_ONLY=${CLUSTER_IP_ONLY:-false} # whether create a 'ClusterIP' type se
 source "${REPO_ROOT}"/hack/util.sh
 
 function usage() {
-  echo "This script will deploy karmada control plane to a given cluster."
-  echo "Usage: hack/deploy-karmada.sh <KUBECONFIG> <CONTEXT_NAME> [KARMADA_API_SERVER_IP]"
-  echo "Example: hack/deploy-karmada.sh ~/.kube/config karmada-host"
+  echo "This script will deploy karmada control plane to a given cluster, it's a high-level script and will be invoked"
+  echo "by the low-level script for a specific scene, such as 'local-up-karmada.sh'. we do not recommend directly use to the end-user"
+  echo "Usage: hack/deploy-karmada.sh <KUBECONFIG> <CONTEXT_NAME> [REMOTE_MODE]"
+  echo "Example: hack/deploy-karmada.sh ~/.kube/config karmada-host false"
+  echo -e "Parameters:\n\tKUBECONFIG\tYour cluster's kubeconfig that you want to install to"
+  echo -e "\tCONTEXT_NAME\tThe name of context in 'kubeconfig'"
+  echo -e "\tREMOTE_MODE\tThis option default is 'false', as that is for the local environment, i.e. for the cluster created by kind."
+  echo -e "\t\t\tAnd if you want to install karmada to a standalone cluster, set it as 'true'. This parameter will decide the way "
+  echo -e "\t\t\tof getting the Karmada API Server' IP"
   unset KUBECONFIG
```

Comment:
Once in the processing of installation, the script will export a new value of KUBECONFIG, I think I should recover the former value of KUBECONFIG instead of removing KUBECONFIG